all-actors/src/main/resources/application.conf

```diff
@@ -117,6 +117,12 @@ certActorSystem {
             nr-of-instances = 5
             dispatcher = cert-dispatcher
           }
+        /TemplateValidateActor
```

Comment:
There are many dispatchers. Pls clean it up and we don't seem to be using anything else other than cert-dispatcher.
The template validation can go to default-dispatcher.

Reply:
done